src/makeRobotAccountant.js

```diff
@@ -6,7 +6,24 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let CALLS_COUNT = 0;
+  const MAX_CALLS_COUNT = 3;
+
+  const ERROR_MESSAGE = 'Bzzz... Error!';
+
+  function getSum(num1) {
+    CALLS_COUNT++;
+
+    return function(num2) {
```

Comment:
not the best idea to use numbers in your variable names,  let's use x and y instead